Clarify usage of "EtherType or Length" argument (#309)
modelica/fmi-ls-bus: docs/4_4_4_ethernet_operations.adoc

| The destination address of the frame.
Only applicable if this is a full Ethernet frame or the first fragment of an Ethernet frame.

| Type or Length
| 2 bytes
| Indicates the type of Ethernet frame (Ethernet II), the TPID (802.1Q) or frame length (802.3).

> _Note: Regarding to <<low-cut-layered-standard-bus-protocol>>, the argument value shall be also stored in `little-endian` byte order._

Reviewer: I think "regarding to" is not correct here, maybe like this:
```suggestion
_Note: As described in <<low-cut-layered-standard-bus-protocol>>, the argument value shall be also stored in `little-endian` byte order._
```

| The destination address of the frame.
Only applicable if this is a full Ethernet frame or the first fragment of an Ethernet frame.

| Type or Length
| 2 bytes

> | Indicates the type of Ethernet frame (Ethernet II), the TPID (802.1Q) or frame length (802.3).

Reviewer: While we're at it, maybe we can add a small remark that this is little endian to prevent confusion.

```suggestion
| Indicates the type of Ethernet frame (Ethernet II), the TPID (802.1Q) or frame length (802.3). Note that this is stored as little-endian.
```

Author: Done

> While we're at it, maybe we can add a small remark that this is little endian to prevent confusion.

@ClemensB done
